### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by mnasyrov** [dumbmatter/fakeIndexedDB#45]
Hello! 

I've got this error during testing a plugin for Dexie.js with FakeIndexedDB, more details about a context here: https://github.com/dfahlander/Dexie.js/issues/1053

Error output:
```
RangeError: Maximum call stack size exceeded

    at FDBTransaction.Object.<anonymous>.FDBTransaction._start
```

If I comment out the following lines in `FDBTransaction.js`, it works (or seems like working):
```js
            // On to the next one
            // if (this._requests.length > 0) {
            //     this._start();
            // }
            // else {
                // Give it another chance for new handlers to be set before finishing
                setImmediate(this._start.bind(this));
            // }
```

**Comment by dumbmatter** [comment]
Thanks for looking into the bug! All the test still pass with your proposed change. And I don't remember why I had those two different ways of calling `this._start`, so maybe there was no good reason in the first place.

**Comment by dumbmatter** [comment]
Fixed in v3.0.2. Thanks again!

**Comment by mnasyrov** [comment]
Thank you for the fix, it works!
